tests/integration/apis/Easing.test.js

```diff
@@ -14,7 +14,7 @@ import { expect } from 'chai';
 describe('Easing', () => {
   const { Easing } = require('react-native');
 
-  it('should have all functions', function () {
+  it('should have all functions', () => {
```

Comment:
Please revert changes like this back. `function` is correct here

Reply:
Arrow functions should be just fine. You're using them above on line 14 already. Is there something I'm missing?

Reply:
Sorry, this change was part of the eslint rule update. I just wholesale applied the rule fixes. I can revert, but do you have more context as to why this one needs to be function?

Reply:
I assumed all were using `function`, in that case this is totally fine